### PR TITLE
Initial add of the scale-demo

### PR DESCRIPTION
**Pull request by brendandburns**
@bgrant0607 for api object style
@kelseyhightower for demo guide

### CODE REVIEW

File: scale-demo/README.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
+# Kubernetes One Million Requests Per Second Demo
+
+## Prerequisites
+### Kubernetes 1.1.x
+This demo requires a Kubernetes 1.1.x cluster, it uses iptables proxy support that was released
+in Kubernetes 1.1.
+
+You can see what version you are running by running:
+
+```console
```
Comment (bgrant0607) [inline scale-demo/README.md]: console format looks for the `$`. Use `shell` if command only.

File: scale-demo/aggregator-rc.yaml [hunk 2]
```diff
@@ -0,0 +1,38 @@
+apiVersion: v1
+kind: ReplicationController
+metadata:
+  labels:
+    app: aggregator
```
Comment (bgrant0607) [inline scale-demo/aggregator-rc.yaml]: Several fields could be set by default, such as these labels.

File: scale-demo/aggregator-rc.yaml [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+    track: alpha
+  name: aggregator
+  namespace: default
```
Comment (bgrant0607) [inline scale-demo/aggregator-rc.yaml]: Could elide.

File: scale-demo/aggregator-rc.yaml [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+spec:
+  replicas: 1
```
Comment (bgrant0607) [inline scale-demo/aggregator-rc.yaml]: Could elide.

File: scale-demo/aggregator-rc.yaml [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+  selector:
+    app: aggregator
```
Comment (bgrant0607) [inline scale-demo/aggregator-rc.yaml]: Could elide.

File: scale-demo/aggregator-rc.yaml [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+    track: alpha
+  template:
+    metadata:
+      name: aggregator
+      namespace: default
```
Comment (bgrant0607) [inline scale-demo/aggregator-rc.yaml]: Could elide.

File: scale-demo/aggregator-rc.yaml [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+  name: aggregator
+  namespace: default
+spec:
+  replicas: 1
+  selector:
+    app: aggregator
+    track: alpha
+  template:
+    metadata:
+      name: aggregator
```
Comment (bgrant0607) [inline scale-demo/aggregator-rc.yaml]: Could elide.

File: scale-demo/aggregator-rc.yaml [hunk 8]
```diff
@@ -0,0 +1,38 @@
...
+      namespace: default
+      labels:
+        app: aggregator
+        track: alpha
+    spec:
+      containers:
+      - command:
+        - /aggregator
```
Comment (bgrant0607) [inline scale-demo/aggregator-rc.yaml]: Could be one line

File: scale-demo/aggregator-rc.yaml [hunk 9]
```diff
@@ -0,0 +1,38 @@
...
+        - -selector=run=vegeta
+        - -use-ip=true
+        - -sleep=1s
+        - -address=0.0.0.0:8080
+        image: gcr.io/google_containers/aggregator:2015-11-10-34b976a
+        imagePullPolicy: IfNotPresent
```
Comment (bgrant0607) [inline scale-demo/aggregator-rc.yaml]: Could be elided.

File: scale-demo/aggregator-rc.yaml [hunk 10]
```diff
@@ -0,0 +1,38 @@
...
+        name: aggregator
```
Comment (bgrant0607) [inline scale-demo/aggregator-rc.yaml]: Probably should be first,

File: scale-demo/aggregator-rc.yaml [hunk 11]
```diff
@@ -0,0 +1,38 @@
...
+        ports:
+        - name: status
```
Comment (bgrant0607) [inline scale-demo/aggregator-rc.yaml]: could be elided

File: scale-demo/aggregator-rc.yaml [hunk 12]
```diff
@@ -0,0 +1,38 @@
...
+          containerPort: 8080
+        resources:
+          limits:
```
Comment (bgrant0607) [inline scale-demo/aggregator-rc.yaml]: requests

File: scale-demo/nginx-rc.yaml [hunk 13]
```diff
@@ -0,0 +1,28 @@
+apiVersion: v1
+kind: ReplicationController
+metadata:
+  labels:
+    run: nginx
```
Comment (bgrant0607) [inline scale-demo/nginx-rc.yaml]: Why not use run?

File: scale-demo/README.md (hunk 1 above)
Comment (brendandburns) [inline scale-demo/README.md]: done.

File: scale-demo/aggregator-rc.yaml (hunk 2 above)
Comment (brendandburns) [inline scale-demo/aggregator-rc.yaml]: done

File: scale-demo/aggregator-rc.yaml (hunk 3 above)
Comment (brendandburns) [inline scale-demo/aggregator-rc.yaml]: done

File: scale-demo/aggregator-rc.yaml (hunk 5 above)
Comment (brendandburns) [inline scale-demo/aggregator-rc.yaml]: done

File: scale-demo/aggregator-rc.yaml (hunk 7 above)
Comment (brendandburns) [inline scale-demo/aggregator-rc.yaml]: done

File: scale-demo/aggregator-rc.yaml (hunk 6 above)
Comment (brendandburns) [inline scale-demo/aggregator-rc.yaml]: done

File: scale-demo/aggregator-rc.yaml (hunk 8 above)
Comment (brendandburns) [inline scale-demo/aggregator-rc.yaml]: done

File: scale-demo/aggregator-rc.yaml (hunk 9 above)
Comment (brendandburns) [inline scale-demo/aggregator-rc.yaml]: done

File: scale-demo/aggregator-rc.yaml (hunk 10 above)
Comment (brendandburns) [inline scale-demo/aggregator-rc.yaml]: done

File: scale-demo/aggregator-rc.yaml (hunk 11 above)
Comment (brendandburns) [inline scale-demo/aggregator-rc.yaml]: done

File: scale-demo/aggregator-rc.yaml (hunk 12 above)
Comment (brendandburns) [inline scale-demo/aggregator-rc.yaml]: done

File: scale-demo/nginx-rc.yaml (hunk 13 above)
Comment (brendandburns) [inline scale-demo/nginx-rc.yaml]: run doesn't allow you to set a custom request CPU.  I could use --override, I suppose, but I think that's uglier.

File: scale-demo/aggregator-rc.yaml (hunk 4 above)
Comment (brendandburns) [inline scale-demo/aggregator-rc.yaml]: done.

**Comment by brendandburns** [comment]
Comments addressed.  I need to add the vegeta godep, I'll do that shortly.

**Comment by bgrant0607** [comment]
Looks fine, thanks.

**Comment by brendandburns** [comment]
merging.

**Pull request merged**